ui_testing/elements.py

```diff
@@ -233,21 +240,27 @@
         'test_unit_btn': {'method': 'tag_name',
                           'value': 'span',
                           'order': 1},
-        'save': {'method': 'class_name',
-                 'value': 'btn-primary',
+        'save': {'method': 'id',
```

Comment:
Changing an old element means u need to run the whole test suite to make sure that u didnt mess another test cases. 

If this element isn't working for your case, Create your element but don't do that unless there r changes in the frontend code.

apply this rule for all your changes in elemens.py